Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla40824.cs

```diff
@@ -5,33 +5,45 @@
 using System.Collections.Generic;
 
 #if UITEST
+using Xamarin.Forms.Core.UITests;
 using Xamarin.UITest;
 using NUnit.Framework;
 #endif
 
 namespace Xamarin.Forms.Controls.Issues
 {
 	[Preserve(AllMembers = true)]
-	[Issue(IssueTracker.Bugzilla, 40824, "ListView item's contextual action menu not being closed upon navigation in AppCompat")]
+	[Issue(IssueTracker.Bugzilla, 40824, "ListView item's contextual action menu not being closed upon navigation in AppCompat"
+		, PlatformAffected.Android)]
+#if UITEST
+	[Category(UITestCategories.ListView)]
+#endif
 	public class Bugzilla40824 : TestContentPage
 	{
+		const string _cat = "Cat";
```

Comment:
`const` fields should use Pascal casing (e.g, `Cat` instead of `_cat`).

Reply:
Sure, I'll revise all of them!